osmaxx/excerptexport/migrations/0021_export_file_format_verbose_name.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9.4 on 2016-04-14 14:23
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('excerptexport', '0020_export_conversion_service_job_id'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='export',
+            name='file_format',
+            field=models.TextField(choices=[('fgdb', 'ESRI File Geodatabase'), ('shapefile', 'ESRI Shapefile'), ('gpkg', 'GeoPackage'), ('spatialite', 'SpatiaLite'), ('garmin', 'Garmin navigation & map data')], verbose_name='file format / data format'),
```

Comment:
Why is this migration separately? Would it make sense join this change into the `0018_export.py`?